src/main/java/jenkins/scm/impl/subversion/SubversionSCMSource.java

```diff
@@ -231,7 +231,7 @@ protected void retrieve(@CheckForNull SCMSourceCriteria criteria,
                             @NonNull final SCMHeadObserver observer,
                             @CheckForNull SCMHeadEvent<?> event,
                             @NonNull TaskListener listener)
-            throws IOException {
+            throws IOException, InterruptedException {
```

Comment:
IMHO not required. I also doubt it is binary compatible for methods, which explicitly call SubversionScmSource instead of the method on the parent class.